src/main/java/picard/fingerprint/CrosscheckReadGroupFingerprints.java

```diff
@@ -60,6 +60,11 @@
             "and print out a library x library matrix with LOD scores.")
     public boolean CROSSCHECK_LIBRARIES = false;
 
+    @Option(doc="Expect all read groups' fingerprints to match, irrespective of their sample names.  By default (with this value set to " +
+            "false), read groups with different sample names are expected to mismatch, and those with the same sample name are expected " +
+            "to match.", mutex = {"EXPECT_ALL_GROUPS_TO_MATCH"})
+    public boolean EXPECT_ALL_READ_GROUPS_TO_MATCH = false;
+
     @Override
     protected String[] customCommandLineValidation() {
```

Comment:
Is there a reason why this method wasn't also moved into `CrosscheckFingerprints`?